### ISSUE TITLE
Package is not available on CRAN

### ISSUE THREAD

**Issue by sclewis23** [halpo/purrrogress#2]
Will this be on CRAN again?

**Comment by jjesusfilho** [comment]
Same question here.

**Comment by jsonProgram** [comment]
1. download the https://cran.r-project.org/src/contrib/Archive/purrrogress/purrrogress_0.1.0.tar.gz file!
like : wget https://cran.r-project.org/src/contrib/Archive/purrrogress/purrrogress_0.1.0.tar.gz

2. R CMD INSTALL purrrogress_0.1.0.tar.gz 
If other packages are missing, go to R and install it(by install.packages).
 
3. retry: R CMD INSTALL purrrogress_0.1.0.tar.gz

**Comment by sclewis23** [comment]
Yes found the archive....was wondering if this package was going to be maintained, and update on CRAN.

**Comment by fghjorth** [comment]
Just checking in on this - big fan of purrrogress, hope to see it back on CRAN!
